stack.yaml

```
@@ -1,4 +1,4 @@
-resolver: lts-15.0
+resolver: lts-15.3
```

Probably should be `15.4`
```suggestion
resolver: lts-15.4
```